tests/lib/nested.sh

```diff
@@ -563,38 +563,42 @@ nested_create_core_vm() {
                     "$TESTSTOOLS"/snaps-state repack_snapd_deb_into_snap snapd "$NESTED_ASSETS_DIR"
                     EXTRA_FUNDAMENTAL="$EXTRA_FUNDAMENTAL --snap $NESTED_ASSETS_DIR/snapd-from-deb.snap"
 
-                    snap download --channel="$CORE_CHANNEL" --basename=core18 core18
-                    repack_core_snap_with_tweaks "core18.snap" "new-core18.snap"
-                    EXTRA_FUNDAMENTAL="$EXTRA_FUNDAMENTAL --snap $PWD/new-core18.snap"
+                    if [ "$NESTED_NO_REPACK_BASE_SNAP" != "true" ]; then
```

Comment:
Could we have $NESTED_REPACK_*_SNAP as default in true in spread.yaml?
And here we check if that value is true

Reply:
sure we could do it that way instead